[2025-06-18 15:00:13,911] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: first_dag.second_task 2021-09-21T00:00:00+00:00 [queued]>
[2025-06-18 15:00:13,938] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: first_dag.second_task 2021-09-21T00:00:00+00:00 [queued]>
[2025-06-18 15:00:13,938] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2025-06-18 15:00:13,939] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2025-06-18 15:00:13,939] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2025-06-18 15:00:14,040] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2021-09-21T00:00:00+00:00
[2025-06-18 15:00:14,056] {standard_task_runner.py:52} INFO - Started process 3534 to run task
[2025-06-18 15:00:14,100] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'first_dag', 'second_task', '2021-09-21T00:00:00+00:00', '--job-id', '9821', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpws4lm9dh', '--error-file', '/tmp/tmph_t016f5']
[2025-06-18 15:00:14,134] {standard_task_runner.py:77} INFO - Job 9821: Subtask second_task
[2025-06-18 15:00:14,377] {logging_mixin.py:104} INFO - Running <TaskInstance: first_dag.second_task 2021-09-21T00:00:00+00:00 [running]> on host 500dc4ac3a72
[2025-06-18 15:00:14,524] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-09-21T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-21T00:00:00+00:00
[2025-06-18 15:00:14,525] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2025-06-18 15:00:14,527] {bash.py:158} INFO - Running command: echo "this is sencond task "
[2025-06-18 15:00:14,572] {bash.py:169} INFO - Output:
[2025-06-18 15:00:14,577] {bash.py:173} INFO - this is sencond task
[2025-06-18 15:00:14,578] {bash.py:177} INFO - Command exited with return code 0
[2025-06-18 15:00:14,724] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=second_task, execution_date=20210921T000000, start_date=20250618T150013, end_date=20250618T150014
[2025-06-18 15:00:14,985] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-18 15:00:15,006] {local_task_job.py:146} INFO - Task exited with return code 0
